### PR TITLE
refac: Move energy requests to views

### PR DESCRIPTION
**Pull request by MWO1024**
# Description

Refactor the solution for energy requests to match the one for wholesale requests. The changes should have no functional impact, aside from utilising another data source.

The streamlining of the solutions should hopefully aid wrt maintenance and the (potential) migration from Wholesale to EDI wrt requests in general.

## Pull-request quality

- [x] The title adheres to [this guide](https://github.com/Mech0z/GitHubGuidelines)
- [x] Tests are written and executed locally
- [ ] Subsystem tests have been tested (by manually deploying to `dev_002`)

### CODE REVIEW

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/AggregatedTimeSeriesQueries.cs [hunk 1]
```diff
@@ -25,43 +27,61 @@ namespace Energinet.DataHub.Wholesale.CalculationResults.Infrastructure.Calculat
...
-                        _whereClauseProvider,
-                        parameters),
-                    parameters.CalculationType)
-                .ConfigureAwait(false);
+        /*
+         * We retain the usage of 'TimeSeriesType' here, to reduce the number of changes required.
+         * However, it is worth noting that the 'TimeSeriesType' is not used in the query per se,
+         * but rather translated to 'MeteringPointType' and 'SettlementMethod',
+         * as these are the proper values used in the raw data (and from the request from EDI).
+         * The reason for this is the existence of logic wrt validation and determination of "all types for an actor role"
```
Comment (lasrinnil) [inline source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/AggregatedTimeSeriesQueries.cs:44]:
```suggestion
         * The reason for this is the existence of logic wrt? validation and determination of "all types for an actor role"
```

Comment (lasrinnil) [reply source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/AggregatedTimeSeriesQueries.cs:44]: what is wrt`?

**Review by lasrinnil — APPROVED** [review]
Nice 🎆

File: source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/AggregatedTimeSeriesQueries.cs (hunk 1 above)
Comment (MWO1024) [reply source/dotnet/wholesale-api/CalculationResults/CalculationResults.Infrastructure/CalculationResults/AggregatedTimeSeriesQueries.cs:44]: 'with regard to'/'with respect to'. A common, at least in math related academia, abbreviation.

**Pull request merged**